src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -141,6 +141,8 @@ private static bool ApplyChangesFromTC33(JObject sarifLog)
                 {
                     // https://github.com/oasis-tcs/sarif-spec/issues/337
                     ConvertToolToDriverInExternalPropertyFiles(run);
+
+                    UpdateAllReportingDescriptorPropertyTypes(run, UpdateReportingDescriptorHelp);
```

Comment:
>run, UpdateReportingDescriptorHelp [](start = 62, length = 34)

Mike, wat do u think about this pattern? i didnt want to duplicate the code which creates the list of paths. I had to rename some stuff, see if it makes sense, thanks :)